Extension/package.json

```diff
@@ -2685,7 +2685,7 @@
       "integrity": "186614524A13EC75BF68798861818D25CE82FDE293C57B725C11F62B0694EC55"
     },
     {
-      "description": "C/C++ language components (OS X)",
+      "description": "C/C++ language components (macOS x86_64)",
```

Comment:
Yeah, I'd say use macOS / x86_64 to be consistent with the other cases.

Reply:
Done